default/stringtables/en.txt

```diff
@@ -7809,10 +7801,23 @@ Direct Weapon
 PC_FIGHTER_HANGAR
 Hangar
 
+PC_FIGHTER_HANGAR_DESC
+'''Hangars are the storage, maintenance, and resupply areas for fighters.
+Since each hangar is built in support of other hangars on the ship, a ship is limited to one specific hangar type.
```

Comment:
Don't need the explanation "Since each hangar ... on the ship,". Would rephrase as just "A ship cannot have more than one type of hangar."